spec/controllers/admin/user_controller_spec.rb

```diff
@@ -0,0 +1,31 @@
+require "rails_helper"
+
+RSpec.describe Admin::UsersController, type: :controller do
+  let!(:user) {FactoryBot.create :user, name: "abc"}
```

Comment:
dữ liệu test nên tạo chuẩn chỉ em nha
```suggestion
  let!(:user) {FactoryBot.create :user, name: "User 1"}
```